charts/operator/templates/aggregate-roles.yaml

```
@@ -0,0 +1,246 @@
+  - watch
+- apiGroups:
+  - osac.openshift.io
+  resources:
+  - tenants/status
+  verbs:
+  - get
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: {{ include "osac-operator.fullname" . }}-admin
```

YAMLlint (1.38.0)

[error] too many spaces inside braces

(braces)